File: src/features/step_definitions/paranormal_steps.js
/*global module:true*/
// steps
var paranormalStepDefinitions = function() {


	this.Given( /^I have 'PhantomJS' on 'Desktop'$/, function( callback ) {
		callback.pending();
	});


	this.When( /^I open the test page$/, function( callback ) {
		callback.pending();
	});


	this.Then( /^I should see '(.*?)' as the page title$/, function( title, callback ) {
		callback.pending();
	});

};

module.exports = paranormalStepDefinitions;

File: src/features/step_definitions/paranormal_steps.rb
# steps

Given /I have paranormal ready/ do
	# NOTHING TO DO HERE, see support/env.rb
end


When /I go to github/ do
	@browser.goto 'https://github.com/bboyle/paranormal'
	assert @browser.title.start_with? 'bboyle/paranormal', 'could not open paranormal repo on github'
end


Then /I should see '(.*)'/ do |title|
	assert @browser.text.include?( title ), 'did not find expected text'
end
